chapters/chapter1.1.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
I think having `.` in file names is not such a great idea. Could you rename this maybe?

`git mv chapter1.1.md chapter1-1.md`

Reply:
Now that I think about it: Let's use more meaningful names. Use the same naming conventions as in slides (just replace `slides` with `chapter`?):

`git mv chapter1.1.md chapter-basics-whatisml.md`